lib/rules/no-unescaped-entities.js

```diff
@@ -59,10 +71,18 @@ module.exports = {
         for (let j = 0; j < entities.length; j++) {
           for (let index = 0; index < rawLine.length; index++) {
             const c = rawLine[index];
-            if (c === entities[j]) {
+            if (typeof entities[j] === 'string') {
```

Comment:
what is this line for? i don't see any new tests.

Reply:
The structure of `DEFAULTS` changed to an array of objects. I'll add some tests.

Reply:
I have added a test which takes an option, `forbid`.